pig-tests/test/org/partiql/pig/tests/PermuteTransformSmokeTests.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package org.partiql.pig.tests
+
+import com.amazon.ionelement.api.ionInt
+import org.junit.jupiter.api.Test
+import org.partiql.pig.tests.generated.ToyLang
+import org.partiql.pig.tests.generated.ToyLangNameless
+import org.partiql.pig.tests.generated.ToyLangToToyLangNamelessVisitorTransform
+import kotlin.test.assertEquals
+
+class PermuteTransformSmokeTests {
```

Comment:
Could add some tests converting back to `ToyLang` and checking for equality with the original. This would probably involve adding 

```
(transform toy_lang_nameless toy_lang)`
```
to `toy-lang.ion`.

Reply:
It's slightly more involved then that since I'd have to include the `name` elements in `toy_lang_nameless`, but sure this is possible.

Next push.